### DISCUSSION TITLE
Embed a Github repository into other GitHub Repositories

### DISCUSSION THREAD

**Discussion by chribonn** [community/community#23950] [New to GitHub]
<p>Hello,</p>
<p>I have a number of Git PowerShell repositories. These all make use of a number of common functions.  Rather than repeat this code (and maintain separate variations) I would like to place this function in a Git Repository on its own and then include this repository within other repositories in which it will be used.</p>
<p>Can I do this?</p>
<p>Thank you</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Submodules should be able to do what you need: <a href="https://git-scm.com/book/en/v2/Git-Tools-Submodules" class="inline-onebox">Git - Submodules</a></p>
